dynawo/sources/Models/Modelica/Dynawo/Examples/Photovoltaics/WECC/PVCurrentSource.mo

```diff
@@ -30,6 +30,8 @@ model PVCurrentSource "WECC PV Model on infinite bus"
     Placement(visible = true, transformation(origin = {80, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 180)));
   Modelica.Blocks.Sources.Constant omegaRefPu(k = 1) annotation(
     Placement(visible = true, transformation(origin = {80, 40}, extent = {{-10, -10}, {10, 10}}, rotation = 180)));
+  Modelica.Blocks.Sources.Constant URefPu(k = 1) annotation(
```

Comment:
I suggest that we test a step in URefPu instead of a constant (from 1 to 1.05pu for instance), and that we check that the voltage follows this reference (for all test cases)

Reply:
Nevermind, not needed